kratos/linear_solvers/amgcl_solver.h

```diff
@@ -101,25 +101,29 @@ AMGCLScalarSolve(
     typename TSparseSpaceType::VectorType& rB,
     typename TSparseSpaceType::IndexType& rIterationNumber,
     double& rResidual,
-    const boost::property_tree::ptree &amgclParams
+    const boost::property_tree::ptree &amgclParams,
+    int verbosity_level
     )
 {
     typedef amgcl::backend::builtin<double> Backend;
 
+    // amgcl::make_solver<
+    //     amgcl::amg<
+    //         Backend,
+    //         amgcl::runtime::coarsening::wrapper,
+    //         amgcl::runtime::relaxation::wrapper
+    //         >,
+    //     amgcl::runtime::solver::wrapper<Backend>
+    //     > solve(amgcl::adapter::zero_copy(TSparseSpaceType::Size1(rA), rA.index1_data().begin(), rA.index2_data().begin(), rA.value_data().begin()), amgclParams);
+
```

Comment:
Personally, I would just remove the commented lines above. Git can track your changes for you.